ingest/scripts/parse_alignment_output.py

```diff
@@ -0,0 +1,44 @@
+"""
+This script parses the results of nextclade alignments to multiple references:
+ - joins results of all alignments
+ - removes rows where there is no alignmentScore (i.e. alignment failed)
+ - group by seqName (=accession) and alignmentScore and keep the row with the highest score
+"""
+
+import click
+import pandas as pd
+
+
+def parse_files(input_files: list[str], output_file: str):
```

Comment:
This does much more than parse which is a bit misleading hence, can call it main_main or something 😄

Reply:
Ah yes kind of silly to create a new function for this I will just move it into main - there is a description of all teh steps at the top of the file :-)